kratos/modeler/modeler.h

```diff
@@ -86,40 +110,72 @@ class Modeler
     ///@name Modeler Stages at Initialize
     ///@{
 
-    /// Import or generate geometry models from external input.
+    /** 
+    * @brief Set up or import the geometry model from external inputs.
+    */
     virtual void SetupGeometryModel()
     {}
 
-    /// Prepare or update the geometry model_part.
+    /** 
+    * @brief Prepare or update the geometry model for further processing. 
+    */
     virtual void PrepareGeometryModel()
     {}
 
-    /// Convert the geometry model or import analysis suitable models.
+    /** 
+    * @brief Configure the geometry into a suitable ModelPart for analysis. 
+    */
     virtual void SetupModelPart()
     {}
 
-    /// This method provides the defaults parameters to avoid conflicts between the different constructors
+    /**
+    * @brief Provides default parameters to ensure consistency across constructors.
+    * @details Derived classes should override this method.
+    * @return Default parameters as a Parameters object.
+    */
     virtual const Parameters GetDefaultParameters() const
     {
-        KRATOS_ERROR << "Calling the base Modeler class GetDefaultParameters. Please implement the GetDefaultParameters in your derived model class." << std::endl;
+        KRATOS_ERROR << "Calling base Modeler class GetDefaultParameters. "
+                    << "Please implement this in your derived class." << std::endl;
         const Parameters default_parameters = Parameters(R"({})");
         return default_parameters;
     }
 
     ///@}
-    ///@name Operators
+    ///@name Mesh and Node Generation
```

Comment:
Shouldn't this be `Operations` according to the rest of files?

Reply:
True

Reply:
(Operators was not right neither)